src/reanimated2/layoutReanimation/web/domUtils.ts

```diff
@@ -131,6 +132,66 @@ export function scheduleAnimationCleanup(
   setTimeout(() => removeWebAnimation(animationName), timeoutValue);
 }
 
+function addChild(child: HTMLElement, parent: Node) {
```

Comment:
maybe reattachChildrenToParent would be a more descriptive name 🤔

Reply:
Well, I agree that we should change this name, though I think yours suggests `child <-> parent` relationship, but in that case it's more general (like attaching grandchild to its grandparent). Anyway, I'm not sure if we care about specific relation in function name. What about something like `reattachElementToAncestor`?

Reply:
Great! 👍